conjur-oss/values.schema.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "required": [
+    "database",
+    "dataKey",
+    "ssl"
```

Comment:
I think database and ssl are conditionally required and as such shouldn't be required in the schema

Reply:
Yes, the `database` and `ssl` sections are optional. For example, if you don't supply any `database.*` values for `helm install`, then the Helm chart will create a database for you with autogenerated SSL certificates & an autogenerated password. Similarly, if you don't supply any `ssl.*` values, the Helm chart will autogenerate SSL certificates and a CA cert chain for Conjur access.

However, there are some "conditionally required" values/fields within the `database` and `ssl` sections. For example, if someone supplies `database.ssl.cert`, then we want to make sure that they also supply `database.ssl.key`. I'll add more comments below for these "conditionally required" fields.